vengine/vengine/application/SceneHandler.cpp

```diff
@@ -67,10 +67,14 @@ void SceneHandler::updateToNextScene()
 void SceneHandler::prepareForRendering()
 {
 	// Update animation timer
-	auto animView = this->scene->getSceneReg().view<AnimationComponent>(entt::exclude<Inactive>);
+	auto animView = this->scene->getSceneReg().view<AnimationComponent, MeshComponent>(entt::exclude<Inactive>);
 	animView.each([&]
-	(AnimationComponent& animationComponent)
+	(AnimationComponent& animationComponent, const MeshComponent& meshComponent)
 		{
+			animationComponent.endTime = 
```

Comment:
Does the animation component even need an endTime if this is the only place using it?